rclpy/CMakeLists.txt

```diff
@@ -185,6 +185,9 @@ clean_windows_flags(_rclpy_pybind11)
 # Extension with code for Action servers and clients
 pybind11_add_module(_rclpy_action SHARED
```

Comment:
Is there an issue with a merge commit? The only two modules should be `_rclpy_pybind11` and `_rclpy_signal_handler` after #743.